specs/predeploys.md

```diff
@@ -48,29 +58,44 @@ The following table includes each of the predeploys. The system version
+| GasPriceOracle                            | 0x420000000000000000000000000000000000000F | Legacy      | No         | Yes     |
+| GovernanceToken                           | 0x4200000000000000000000000000000000000042 | Legacy      | No         | No      |
+| L1Block                                   | 0x4200000000000000000000000000000000000015 | Bedrock     | No         | Yes     |
+| L2ToL1MessagePasser                       | 0x4200000000000000000000000000000000000016 | Bedrock     | No         | Yes     |
+| L2ERC721Bridge                            | 0x4200000000000000000000000000000000000014 | Legacy      | No         | Yes     |
+| OptimismMintableERC721Factory             | 0x4200000000000000000000000000000000000017 | Bedrock     | No         | Yes     |
+| ProxyAdmin                                | 0x4200000000000000000000000000000000000018 | Bedrock     | No         | Yes     |
+| BaseFeeVault                              | 0x4200000000000000000000000000000000000019 | Bedrock     | No         | Yes     |
+| L1FeeVault                                | 0x420000000000000000000000000000000000001a | Bedrock     | No         | Yes     |
+| SchemaRegistry                            | 0x4200000000000000000000000000000000000020 | Bedrock     | No         | Yes     |
+| EAS                                       | 0x4200000000000000000000000000000000000021 | Bedrock     | No         | Yes     |
+| create2Deployer                           | 0x13b0D85CcB8bf860b6b79AF3029fCA081AE9beF2 | Canyon      | No         | No      |
+| Safe                                      | 0x69f4D1788e39c87893C980c06EdF4b7f686e2938 | New Chains* | No         | No      |
+| SafeL2                                    | 0xfb1bffC9d739B8D520DaF37dF666da4C687191EA | New Chains* | No         | No      |
+| MultiSend                                 | 0x998739BFdAAdde7C933B942a68053933098f9EDa | New Chains* | No         | No      |
+| MultiSendCallOnly                         | 0xA1dabEF33b3B82c7814B6D82A79e50F4AC44102B | New Chains* | No         | No      |
+| SafeSingletonFactory                      | 0x914d7Fec6aaC8cd542e72Bca78B30650d45643d7 | New Chains* | No         | No      |
+| MultiCall3                                | 0xcA11bde05977b3631167028862bE2a173976CA11 | New Chains* | No         | No      |
+| Arachnid's Deterministic Deployment Proxy | 0x4e59b44847b379578588920cA78FbF26c0B4956C | New Chains* | No         | No      |
+| Permit2                                   | 0x000000000022D473030F116dDEE9F6B43aC78BA3 | New Chains* | No         | No      |
+| ERC-4337 EntryPoint                       | 0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789 | New Chains* | No         | No      |
+| ERC-4337 SenderCreator                    | 0x7fc98430eaedbb6070b35b39d798725049088348 | New Chains* | No         | No      |
+
+\* These predeploys were added to the L2 genesis, so only new chains started with the
```

Comment:
We should remove this sort of language from the spec "working on...". It will fall out of date quickly and also isn't the type of language that is normally found in a technical specification. Perhaps we just say bedrock with a `*` and then in the description we say that early bedrock chains did not include them in genesis but all chains going forward will have them in genesis